.github/ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,25 @@
-Describe in one or two sentences why you're opening this issue:
+### Description:
+<!-- Describe in one or two sentences why you're opening this issue: -->
+> **REPLACE-ME**
 
+### Tell us a bit about yourself:
+* Version of JekyllAdmin I'm using <HINT: use `bundle show` to check>: ***REPLACE-ME***
```

Comment:
I wanted the issue drafter to insert the info exactly between the three stars to have their response formatted. Rarely does a user style their issue draft.
I'll remove those too..